code/modules/research/designs/limbgrower_designs.dm

```diff
@@ -8,31 +8,31 @@
 	build_type = LIMBGROWER
 	reagents_list = list("synthflesh" = 25)
 	build_path = /obj/item/bodypart/l_arm
-	category = list("initial","human","lizard","plasmaman")
+	category = list("initial","human","lizard","plasmaman", "ethari")
 
 /datum/design/rightarm
 	name = "Right Arm"
 	id = "rightarm"
 	build_type = LIMBGROWER
 	reagents_list = list("synthflesh" = 25)
 	build_path = /obj/item/bodypart/r_arm
-	category = list("initial","human","lizard","plasmaman")
+	category = list("initial","human","lizard","plasmaman", "ethari")
 
 /datum/design/leftleg
 	name = "Left Leg"
 	id = "leftleg"
 	build_type = LIMBGROWER
 	reagents_list = list("synthflesh" = 25)
 	build_path = /obj/item/bodypart/l_leg
-	category = list("initial","human","lizard","plasmaman")
+	category = list("initial","human","lizard","plasmaman", "ethari")
 
 /datum/design/rightleg
 	name = "Right Leg"
 	id = "rightleg"
 	build_type = LIMBGROWER
 	reagents_list = list("synthflesh" = 25)
 	build_path = /obj/item/bodypart/r_leg
-	category = list("initial","human","lizard","plasmaman")
+	category = list("initial","human","lizard","plasmaman", "ethari")
```

Comment:
Please make the argument spacing consistent.

Reply:
Also while these appear to be Ethari bug fixes, shouldn't the lizard's be unathi?